src/JSHOP2/InternalDomain.java

```diff
@@ -594,6 +597,13 @@ public String generateProblemCode(LinkedList<Vector<Predicate>> states, LinkedLi
       //-- For each predicate, in the initial world state of the problem
       for (Predicate p : state)
       {
+        if(counter++ == 500){
```

Comment:
The surrounding code uses spaces in the following situations. Thus, I suggest to stay consistent here and in the following lines:
1. After keywords
2. Before opening braces
3. Around binary operators
4. Around assignment equals symbols

Additionally, a line of comment for this block may be helpful. The content can e.g. be similar to your message in the pull request.